.flake8

```diff
@@ -2,3 +2,4 @@
 max-line-length = 88
 extend-ignore = E501
 exclude = .venv
+ignore = E203, W503
```

Comment:
Heya! Can you share the error you were getting with this? I can't seem to reproduce it. Thanks

Reply:
If I take out the line and run flake8 at the root level I get:
`./code/backend/batch/utilities/helpers/embedders/push_embedder.py:87:46: E203 whitespace before ':'`

Then if I add the ignore for E203 I get many errors like:
`./code/backend/batch/utilities/common/answer.py:27:13: W503 line break before binary operator`

I don't get functional errors, just in flake, but am not able to commit with those errors.  And if I try to adjust the code to pass flake, black reformats it back during the commit.

Reply:
Gotcha! Not sure what was going on last week, but can now reproduce this. Looks like it's a known issue and documented here https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8